vhost-device-scmi/src/devices/iio.rs

```diff
@@ -453,8 +673,12 @@ impl IIOSensor {
 
     fn read_axis(&self, axis: &Axis) -> Result<i64, ScmiDeviceError> {
         let path_result = axis.path.clone().into_string();
-        let mut value: i64 =
+        let value: i64 =
             read_number_from_file(Path::new(&(path_result.unwrap() + "_raw")))?.unwrap();
+        self.deal_axis_raw_data(value, axis)
+    }
+
+    fn deal_axis_raw_data(&self, mut value: i64, axis: &Axis) -> Result<i64, ScmiDeviceError> {
```

Comment:
Can you do this kind of refactoring in a separate commit?
This patch is very big, so please try to split at least the refactoring in other patches.

Reply:
Oh...Yes, this patch is very big. I will try to separate it in next push.

Reply:
I have separated it into two patches